src/test/ts/alien/Loader.ts

```diff
@@ -19,40 +18,39 @@ const getRoot = () => SelectorFind.descendant(SugarBody.body(), '#root').getOrTh
   return root;
 });
 
-const cRender = (data: Record<string, any> = {}, template: string = `<editor :init="init" ></editor>`) =>
-  Chain.async<Context, Context>((_value, next, _die) => {
-    const root = getRoot();
-    const mountPoint = SugarElement.fromTag('div');
-    Insert.append(root, mountPoint);
-
-    const originalInit = data.init || {};
-    const originalSetup = originalInit.setup || Fun.noop;
-
-    const vm = createApp({
-      template,
-      components: {
-        Editor
-      },
-      data: () => ({
-        ...data,
-        outputFormat: 'text',
-        init: {
-          ...originalInit,
-          setup: (editor: any) => {
-            originalSetup(editor);
-            editor.on('SkinLoaded', () => {
-              setTimeout(() => {
-                next({ editor, vm });
-              }, 0);
-            });
-          }
+const pRender = (data: Record<string, any> = {}, template: string = `<editor :init="init"></editor>`): Promise<any> => new Promise((resolve) => {
```

Comment:
```suggestion
const pRender = (data: Record<string, any> = {}, template: string = `<editor :init="init"></editor>`): Promise<Record<string, any> => new Promise((resolve) => {
```

Reply:
Update in [this](https://github.com/tinymce/tinymce-vue/pull/434/commits/d5b7244dcb95121f0af5989c16c8fb6d2cecf822) commit and retested to verify and adding missing `>`